[IMP] payroll: updating payslips info and pics for 17

larm-odoo opened the pull request:
New UI and new instructions, so doc needs to be updated for V17.

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/7740).

jero-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,177 @@ from this dashboard.
+   contact information. If there is no bank information, a payslip cannot be paid, and an error
+   appears when the :guilabel:`Make Payment` button is clicked. Banking information can be found in
+   the :ref:`Private Information <employees/private-info>` tab on the employee's card. Edit the
+   employee card, and add banking information, if it is missing.
```
```suggestion
   employee card, and add banking information if it is missing.
```

jero-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -190,22 +210,20 @@ Click on an individual payslip to view the details for that payslip on a separat
-icon) to export all payslips to an Excel spreadsheet.
+regardless of whether some are selected or not. Click on the :guilabel:`⚙️ (gear)` icon at the end
+of the words `Employee Payslips` in the top-left corner. This reveals a drop-down menu. Click
+:guilabel:`Export All` to export all payslips to an Excel spreadsheet.
```
```suggestion
:guilabel:`Export All` to export all payslips to a spreadsheet.
```

jero-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,144 +239,101 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
+
+Next, select the date range to which the batch applies. Click into one of the :guilabel:`Period`
+fields and a calendar pop-up window appears. From this calendar pop-up window, navigate to the
+correct month, and click on the corresponding day for both the start and end dates of the batch.
```
```suggestion
correct month and click on the corresponding day for both the start and end dates of the batch.
```

jero-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,144 +239,101 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
-:guilabel:`Period` fields, which reveals a calendar pop-up window. From this calendar pop-up window,
-navigate to the correct month, and click on the corresponding day for both the start and end dates.
+The current company populates the :guilabel:`Company` field. If operating in a multi-company
+environment, it is not possible to modify the :guilabel:`Company` from the form. The batch must be
```
```suggestion
environment, it is **not** possible to modify the :guilabel:`Company` from the form. The batch must be
```

jero-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,144 +239,101 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
+- :guilabel:`New`: batches without any payslips added to them have a status of :guilabel:`New`.
+  Only these batches present the option to add payslips to the selected batch. Click the
+  :guilabel:`Add Payslips` button and an :guilabel:`Add Payslips` pop-up window appears. Only
+  payslips that are able to be added to the batch (payslips not currently part of a batch) appear on
```
```suggestion
  payslips that can be added to the batch (payslips not currently part of a batch) appear on
```

jero-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,144 +239,101 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
+  - :guilabel:`Create Payment Report`: click the :guilabel:`Create Payment Report` button and a
+    :guilabel:`Select a bank journal` pop-up window appears. Select the correct bank journal from
+    the drop-down menu. The batch name appears in the :guilabel:`File name` field, but this can be
+    modified, if desired. Finally, click :guilabel:`Confirm` to process the payslips, and pay the
```
```suggestion
    modified if desired. Finally, click :guilabel:`Confirm` to process the payslips and pay the
```

jero-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,144 +239,101 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
 
-From here, click the :guilabel:`Generate Draft Entry` button to change the payslips' status from
-:guilabel:`Draft` to :guilabel:`Done`.
+Use the breadcrumb menu to navigate back to the individual batch detail page, or back to the list of
```
```suggestion
Use the breadcrumb menu to navigate back to the individual batch detail page or back to the list of
```

jero-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,144 +239,101 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
-page, click on the desired batch (or batches) to create commissions payslips for, then click the
-:guilabel:`Generate Commission Payslips` button.
+First, select the desired batches by clicking the box to the left of each batch to create
+commissions payslips for. Next, click the :guilabel:`Generate Warrant Payslips` button at the top of
```
```suggestion
commission payslips for. Next, click the :guilabel:`Generate Warrant Payslips` button at the top of
```

jero-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,177 @@ from this dashboard.
+
+  Then, click on the desired date to select it.
+- :guilabel:`Date Account`: located in the :guilabel:`Other Info` tab, enter the date on
+  which the payslip should be posted.
```
early line break, move 'which' to previous line

jero-odoo reviewed (changes requested):
Hi @larm-odoo all set, let me know if you have any questions, or when you are ready for me to take another look 👍

larm-odoo commented:
All set for you @jero-odoo!

larm-odoo commented:
Hi @ksc-odoo - this is ready for a final review! JERO did the first round, and I used google docs, grammarly, and ChatGPT for the 2nd round. All set!

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,178 @@ from this dashboard.
 To view all payslips regardless of status, go to :menuselection:`Payroll app --> Payslips --> All
-Payslips`. Here, all payslips are organized by batch (in a default list view).
+Payslips`. The :guilabel:`Employee Payslips` page loads, displaying all payslips, organized by batch
+(in a default list view).
```
```suggestion
in a default nested list view.
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,178 @@ from this dashboard.
-appears on the far-right side, indicating if it is in :guilabel:`Draft Mode`, :guilabel:`Waiting`,
-or if it is :guilabel:`Done`.
+The number of payslips in the batch is written in parenthesis after the batch name. The
+:guilabel:`Status` for each individual payslip appears on the far-right side, indicating if it is in
+:guilabel:`Draft Mode`, :guilabel:`Waiting`, :guilabel:`Done`, or if it is :guilabel:`Paid`.
```
```suggestion
:guilabel:`Status` for each individual payslip appears on the far-right side, indicating one of the
following status options: :guilabel:`Draft`, :guilabel:`Waiting`, :guilabel:`Done`, or
:guilabel:`Paid`.
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,178 @@ from this dashboard.
    In order for a payslip to be paid, the employee *must* have a bank account entered in their
-   contact information. If there is no bank information, a payslip cannot be paid, and an error will
-   appear when the :guilabel:`Make Payment` button is clicked. Banking information can be found in
-   the :guilabel:`Private Information` tab on the employee's card. Edit the employee card, and add
-   banking information, if it is missing.
+   contact information. If there is no bank information, a payslip cannot be paid, and an error
+   appears when the :guilabel:`Make Payment` button is clicked. Banking information can be found in
+   the :ref:`Private Information <employees/private-info>` tab on the employee's card. Edit the
+   employee card, and add banking information if it is missing.
```
I'm assuming this is why I continued to get that error...? not sure. If so, since I encountered that issue after click 'Register Payment' this admonition may be best if placed right after that step.

larm-odoo replied:
I agree! That ight have been why- this has been moved up.

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,178 @@ from this dashboard.
 .. note::
    - :guilabel:`Draft` indicates the payslip is created, and there is still time to make edits,
      since the amounts are not calculated.
-   - `Waiting` indicates the payslip has been calculated and the salary details can be found in the
-     *Salary Computation* tab.
-   - `Done` indicates the payslip is calculated and ready to be paid.
+   - :guilabel:`Waiting` indicates the payslip has been calculated and the salary details can be
+     found in the *Salary Computation* tab.
+   - :guilabel:`Done` indicates the payslip is calculated and ready to be paid.
+   - :guilabel:`Paid` indicates the employee has been paid.
```
I think this can work as a list without being an admonition -- what do you think?

also, I'd suggest using colons to define each status.

For example:

'Draft': indicated the payslip is ...

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -190,22 +211,20 @@ Click on an individual payslip to view the details for that payslip on a separat
+:guilabel:`New` button in the upper-left corner. Doing so reveals a separate blank payslip form
 page. On that blank payslip form page, enter all the necessary information, as described in the
-:ref:`Create a new payslip <payroll/new-payslip>` section.
+:ref:`Create new payslip <payroll/new-payslip>` section.
```
```suggestion
:ref:`Create new payslips <payroll/new-payslip>` section.
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -190,22 +211,20 @@ Click on an individual payslip to view the details for that payslip on a separat
-   :align: center
-   :alt: Click on the Print smart button to print payslips to a PDF.
+Payslips` pages, first select the desired payslips by clicking on the individual checkbox to the
+left of each payslip to be printed. Or, click the box to the left of :guilabel:`Reference`, which
```
```suggestion
left of each payslip to be printed. Or, click the box to the left of the :guilabel:`Reference` column title, which
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -190,22 +211,20 @@ Click on an individual payslip to view the details for that payslip on a separat
+selects all visible payslips on the page. Then, click the :guilabel:`Print` button to print the
+payslips.
 
 Payslips can also be exported to an Excel spreadsheet. When exporting, all payslips are exported
```
```suggestion
Payslips can also be exported to an Excel spreadsheet. When exporting, *all* payslips are exported,
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -190,22 +211,20 @@ Click on an individual payslip to view the details for that payslip on a separat
 Payslips can also be exported to an Excel spreadsheet. When exporting, all payslips are exported
-regardless of whether some are selected or not. Click on the :guilabel:`Export All` button (download
-icon) to export all payslips to an Excel spreadsheet.
+regardless of whether some are selected or not. Click on the :guilabel:`⚙️ (gear)` icon at the end
+of the words `Employee Payslips` in the top-left corner. This reveals a drop-down menu. Click
+:guilabel:`Export All` to export all payslips to a spreadsheet.
```
```suggestion
regardless of whether some are selected or not. To export payslips, click on the :guilabel:`⚙️ (gear)` icon at the end
of the words `Employee Payslips` in the top-left corner. This reveals a drop-down menu. Click
:guilabel:`Export All` to export all payslips to a spreadsheet.
```
when certain payslips are selected, the actions gear icon appears at the top-center of the screen, and isn't referred to as 'Export' all...it's just 'Export' -- it may be beneficial to highlight this potential difference, as well

larm-odoo replied:
I missed that- I added both options, thank you!

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,144 +240,100 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
-to which the batch applies, by clicking the :guilabel:`▼ (drop-down arrow)` icon in the
-:guilabel:`Period` fields, which reveals a calendar pop-up window. From this calendar pop-up window,
-navigate to the correct month, and click on the corresponding day for both the start and end dates.
+On the new payslip batch form, enter the :guilabel:`Batch Name` in the field.
```
```suggestion
On the new payslip batch form, enter the :guilabel:`Batch Name`.
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,144 +240,100 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
-   :alt: Enter the details for the new batch.
+Next, select the date range to which the batch applies. Click into one of the :guilabel:`Period`
+fields and a calendar pop-up window appears. From this calendar pop-up window, navigate to the
+correct month and click on the corresponding day for both the start and end dates of the batch.
```
```suggestion
correct month, and click on the corresponding day for both the start and end dates of the batch.
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,144 +240,100 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
-:guilabel:`Generation Date` field, select the date the payslips should be generated from a calendar
-pop-up window. This generated date is reflected in the accounting journal entries.
+The current company populates the :guilabel:`Company` field. If operating in a multi-company
+environment, it is **not** possible to modify the :guilabel:`Company` from the form. The batch must
```
```suggestion
environment, it is **not** possible to modify the :guilabel:`Company` from the form. The batch **must**
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,144 +240,100 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
-If there are any errors or issues preventing the payslips from being generated, an error message
-appears in the top-right section. This error box disappears on its own after several seconds, or the
-:guilabel:`✖ (x mark)` icon can be clicked to close the alert.
+Click the checkbox to the left of each payslip to select it. When all desired payslips have been
```
```suggestion
Click the checkbox to the left of each desired payslip to select it. When all desired payslips have been
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,144 +240,100 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
+Click on an individual batch to view the details for that batch on a separate page. On this batch
+detail page, different options (buttons) appear at the top, depending on the status of the batch:
+
+- :guilabel:`New`: batches without any payslips added to them have a status of :guilabel:`New`.
```
is this to represent the New button or New status?

larm-odoo replied:
I totally redid this section so it is WAY more clear! I can see how confusing this was to read now that I am looking at it a third time!

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,144 +240,100 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
+detail page, different options (buttons) appear at the top, depending on the status of the batch:
+
+- :guilabel:`New`: batches without any payslips added to them have a status of :guilabel:`New`.
+  Only these batches present the option to add payslips to the selected batch. Click the
```
```suggestion
  Only these batches present the option to add payslips to the selected batch, via the :guilabel:`Add Payslips` button. Click the
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,144 +240,100 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
+  Only these batches present the option to add payslips to the selected batch. Click the
+  :guilabel:`Add Payslips` button and an :guilabel:`Add Payslips` pop-up window appears. Only
+  payslips that can be added to the batch (payslips not currently part of a batch) appear on the
+  list. Select the desired payslips by clicking the small box to the left of each payslip name, then
```
```suggestion
  list. Select the desired payslips by clicking the checkbox to the left of each payslip name, then
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,144 +240,100 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
+  list. Select the desired payslips by clicking the small box to the left of each payslip name, then
+  click the :guilabel:`Select` button to add them to the batch.
+- :guilabel:`Confirmed`: batches that have been created and have payslips in them, but the payslips
+  have not been paid out yet, have a status of :guilabel:`Confirmed`. These batches display a
```
```suggestion
  have *not* been paid out yet, have a status of :guilabel:`Confirmed`. These batches display a
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,144 +240,100 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
-:guilabel:`Draft` to :guilabel:`Done`.
+Click on an individual batch to view the details for that batch on a separate page. On this batch
+detail page, different options (buttons) appear at the top, depending on the status of the batch:
+
```
I think a line here introducing the following list as 'Statuses' would be very helpful

larm-odoo replied:
With the way I redid this, I think it is more clear- but I am open to more suggestions after your second look though!

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,144 +240,100 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
+  - :guilabel:`Create Payment Report`: click the :guilabel:`Create Payment Report` button and a
+    :guilabel:`Select a bank journal` pop-up window appears. Select the correct bank journal from
+    the drop-down menu. The batch name appears in the :guilabel:`File name` field, but this can be
+    modified if desired. Finally, click :guilabel:`Confirm` to process the payslips and pay the
```
```suggestion
    modified, if desired. Finally, click :guilabel:`Confirm` to process the payslips, and pay the
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,144 +240,100 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
+  - :guilabel:`Mark as paid`: after the payments have been created via the :guilabel:`Create Payment
+    Report` button, the payslips need to be marked as paid in the database. Click the
+    :guilabel:`Mark as paid` button, and the status of the batch changes to :guilabel:`Paid`.
+
```
I'm also seeing a 'Create ABA File' button and 'Set to Draft' button -- do you think those need to be explained?

larm-odoo replied:
I added the set to draft option where it appears- but I never see the ABA one- I think that is for specific accounts with some kind of ABA setting on them and not a 'universal' issue- I am checking with the team and seeing if this needs to be added/explained/is common.

larm-odoo replied:
OK- these are form specific localization settings and are not universal! =)  For example, SEPA is in Europe only.

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,144 +240,100 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
+:guilabel:`Payslips` smart button, located above the batch information, in the center. Click the
+:guilabel:`Payslips` smart button to view a list of the individual payslips.
+
+Use the breadcrumb menu to navigate back to the individual batch detail page or back to the list of
```
```suggestion
Use the breadcrumb menu to navigate back to the individual batch detail page, or back to the list of
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,144 +240,100 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
-(:menuselection:`Payroll app --> Payslips --> Batches`). To generate commission payslips from this
-page, click on the desired batch (or batches) to create commissions payslips for, then click the
-:guilabel:`Generate Commission Payslips` button.
+First, select the desired batches by clicking the box to the left of each batch to create commission
+payslips for. Next, click the :guilabel:`Generate Warrant Payslips` button at the top of the page.
```
```suggestion
First, select the desired batches by clicking the box to the left of each batch for which commission
payslips should be created. Next, click the :guilabel:`Generate Warrant Payslips` button at the top
of the page.
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,144 +240,100 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
+In this pop-up window, click on the drop-down menus, located beside the :guilabel:`Period` field, to
 reveal calendar pop-up windows. On these calendar pop-up windows, select the desired period for
 which the payslips are being generated. Using the :guilabel:`< (left)` and :guilabel:`> (right)`
 arrows, navigate to the correct month, and click on the date to select it.
```
```suggestion
arrows to navigate to the correct month, and click on the date to select it.
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -369,18 +344,17 @@ When a department is selected, the employees listed for that department appear i
 
 Under the :guilabel:`Employee` section, enter the :guilabel:`Commission Amount` for each employee in
-the appropriate column. To remove an employee, click the :guilabel:`🗑️ (trash)` icon to remove the
+the far right column. To remove an employee, click the :guilabel:`🗑️ (trash)` icon to remove the
```
```suggestion
the far-right column. To remove an employee, click the :guilabel:`🗑️ (trash)` icon to remove the
```

ksc-odoo reviewed (changes requested):
hey @larm-odoo -- just finished Final Reviewing this PR. Once you address my feedback, tag me again for another look. Thanks!

larm-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,178 @@ from this dashboard.
 
-Optional fields
-~~~~~~~~~~~~~~~
+Other fields
+~~~~~~~~~~~~
```
Good idea- that's what I did on previous forms, where there was SO MUCH to enter. I had a 'required fields' and an 'optional fields' section. But I agree- I remember doing that with a different form, and saying 'this field is required. for all the parts that did that.

larm-odoo commented on content/applications/hr/employees/new_employee.rst:
```diff
@@ -347,19 +347,40 @@ information for the new employee:
-     roles.
+   If an employee's working hours are not configured as a working time for the company, new working
+   times can be added, or existing working times can be modified. To add or modify a working time,
+   go to the :menuselection:`Payroll app --> Configuration --> Working Times`, and add a new working
```
A better solution: I linked it to the Working Schedules anchor in the other Payroll config doc! This way, all the steps are there and I won't make this doc longer than it is.  =) 

Also, you are correct- they interchange working times and schedules- you go to the schedules and it lists the times- that tripped me up!

larm-odoo commented on content/applications/hr/employees/new_employee.rst:
```diff
@@ -347,19 +347,40 @@ information for the new employee:
-     If this field is blank, they are assigned shifts, regardless of the role. The
-     :guilabel:`Default Role` has precedence over the other roles when assigning shifts.
+   After the new working time is created, or an existing one is modified, select the employee's
+   working hours using the drop-down menu in the :guilabel:`Schedule` section of the
+   :guilabel:`Work Information` tab.
```
I thought it was clear since this doc is the 'new employee' doc, so it was understood that it was on the new employee form- but I hear ya! I changed it up, so it's more clear. I think it should work now!

larm-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,178 @@ from this dashboard.
+
+  Repeat this process to modify the end date for the payslip.
+- :guilabel:`Contract`: using the drop-down menu, select the desired contract for the employee. Only
+  the available corresponding contracts for the selected employee appear as options.
```
I made a note in the Employee part- since people should ONLY be writing pay slips for employees in the system (doesn't make sense to not follow that order- they are hired, added to the system, with a contract, then after they have worked, they get a paycheck. Agree? Made sense to me!) I think the note should clear it up, and it links to the doc explaining how to do all that.

larm-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,178 @@ from this dashboard.
+- :guilabel:`Contract`: using the drop-down menu, select the desired contract for the employee. Only
+  the available corresponding contracts for the selected employee appear as options.
+- :guilabel:`Structure`: using the drop-down menu, select the salary structure type. Only the
+  corresponding structures associated with the selected contract for the employee appear as options.
```
Nice catch! I clarified that all structures appear, and if structures do not apply for a specific employee or contract, it doesn't appear.

larm-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,178 @@ from this dashboard.
+  Then, click on the desired date to select it.
+- :guilabel:`Date Account`: located in the :guilabel:`Other Info` tab, enter the date on which the
+  payslip should be posted.
+- :guilabel:`Accounting Entry`: this field, located in the :guilabel:`Accounting Information` tab,
```
Again- good catch!

larm-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,178 @@ from this dashboard.
+
+When all the necessary information on the payslip is entered, click the :guilabel:`Compute Sheet`
+button. All the information on the payslip is saved, and the :guilabel:`Salary Computation` tab
+auto-populates based on the information on the employee's contract or attendance records.
```
No- this is a 'you have to press this button to make this other information appear' situation. It won't compute/populate that tab until that button is pressed.

larm-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,178 @@ from this dashboard.
 
-To print the payslip, click the :guilabel:`Print` button. To cancel the payslip, click the
-:guilabel:`Cancel Payslip` button.
+The chatter is automatically updated to show the email sent to the employee, along with a PDF copy
+of the payslip.
```
When I did this, it took a full minute to "think it through" so to speak! A new tab opens, where I assume the PDF is being made and downloaded- then the tab disappears (weird). BUT after that, it does appear- but you may need to refresh the window to see it. This is what happens after I did the same thing, waited for that tab, an refreshed:
![image](https://github.com/odoo/documentation/assets/121518652/575bf24c-c338-429b-860a-1ce75e6ae46a)


I've added more info on this so it's more clear!

larm-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,178 @@ from this dashboard.
    :alt: The new payslip is emailed to the employee and the email appears in the chatter.
 
-Next, the payment must be sent to the employee. To do this, click the :guilabel:`Make Payment`
+Next, the payment must be sent to the employee. To do this, click the :guilabel:`Register Payment`
```
I get that a lot- it is usually due to a setting on the salary rules, or outside a contract (those are the most common errors I get). I tried again in a runbot, and got many successful payments- so I am wondering if someone went into a runbot you were using and changed something in the salary configurator, or rules, etc that caused a cascading issue? It's easy to totally derail the whole Payroll system by changing something somewhere else =). 

I added a tip block explaining errors, and warnings- where to find them, etc. 

So for the next look through, if you have multiple issues again- I'd honestly try a different runbot and see if that helps! I did get that problem half the time when writing this doc, and trying different runbots helped.

larm-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,178 @@ from this dashboard.
-   :alt: Click Make Payment to send the payment to the employee.
+Odoo automatically checks bank account information. If there is an error with the employee's listed
+bank account, an error appears in a pop-up window, stating, `The employee bank account is
+untrusted.` If this error appears, update the employee's bank account information on their
+:ref:`Employee Form <employees/private-info>`.
 
-If a payment needs to be cancelled or refunded, click the corresponding :guilabel:`Refund` or
-:guilabel:`Cancel Payslip` button, located at the top of the payslip form.
+If a payment needs to be canceled or refunded, click the corresponding :guilabel:`Cancel` or
+:guilabel:`Refund` button, located at the top-left of the screen.
 
 .. important::
    In order for a payslip to be paid, the employee *must* have a bank account entered in their
-   contact information. If there is no bank information, a payslip cannot be paid, and an error will
-   appear when the :guilabel:`Make Payment` button is clicked. Banking information can be found in
-   the :guilabel:`Private Information` tab on the employee's card. Edit the employee card, and add
-   banking information, if it is missing.
+   contact information. If there is no bank information, a payslip cannot be paid, and an error
+   appears when the :guilabel:`Make Payment` button is clicked. Banking information can be found in
+   the :ref:`Private Information <employees/private-info>` tab on the employee's card. Edit the
+   employee card, and add banking information if it is missing.
 
    .. image:: payslips/banking.png
       :align: center
       :alt: Banking information can be entered in an employee's card.
 
+.. _payroll/all-payslips:
```
I totally understand! And like my comment from above, I would try a different runbot i you get a big issue when reviewing. People can easily 'break' the various threads connecting payroll to everything else, and sometimes, just changing a rule or deleting something in the settings, can completely break everything. I know I broke it several times =D

larm-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,178 @@ from this dashboard.
-
-When all the necessary information on the payslip is entered, click :guilabel:`Save` to save the
-data, or click :guilabel:`Discard` to delete the entry.
+- :guilabel:`Has Negative Net To Report`: click the checkbox if the employee has a negative net
+  amount for this payslip.
```
This only shows up if it applies- I added this.

larm-odoo commented:
Thank you for such a thorough review, @ksc-odoo! I addressed all your questions and replied with any explanations needed. I believe I clarified any and all confusing areas, and expanded the section in batches under statuses and really went through and added pics for each of the statuses as well. When you test in runbot, there are a lot of issues that can happen because of localizations, and things not being "real" bank accounts, and errors happening. I go into the settings menu and turn off SEPA payments- this helps a lot in Runbot. Otherwise, I try different runbots if there are lots of issues in testing the flow. I find half the time, a runbot is totally unusable (too many settings changed or messed with by other users) and the other half, works fairly well with minimal tweaking. But if at any point along the process you have any issues, please let me know so I can jump in and explain something/show a setting/etc. There's just WAY too many variables to add instructions for every configuration, so I was trying to keep it relatively universal and brief (laughable, me writing a "brief" doc, I now!) This is ready for a second look!

Also, sorry I also edited the employees doc- that threw a ton of extra revisions as well. When I was in there adding the custom anchors I needed, I also fixed things the linter flagged- and I couldn't leave well enough alone =)

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,235 @@ from this dashboard.
+   :align: center
+   :alt: The fields filled out in the worked days and inputs tab.
+
+Salary computation tab
```
```suggestion
Salary Computation tab
```
same capitalization methodology as mentioned above.

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,235 @@ from this dashboard.
+**********************
+
+- :guilabel:`Salary Computation`: the :guilabel:`Salary Computation` tab is automatically filled in
+  after the :guilabel:`Compute Sheet` button is clicked. Doing so displays the entry's wages,
```
```suggestion
  after the :guilabel:`Compute Sheet` button is clicked. Doing so displays the wages,
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,235 @@ from this dashboard.
+   :align: center
+   :alt: The fields filled out in the salary computation tab.
+
+Other info tab
```
```suggestion
Other Info tab
```
same reasoning as above

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,235 @@ from this dashboard.
+Other info tab
+**************
+
+- :guilabel:`Payslip Name`: in the :guilabel:`Other Info` tab, type in a name for the payslip in
```
```suggestion
- :guilabel:`Payslip Name`: type in a name for the payslip in
```
since it's already in the Other Info tab section, I don't believe the second mention of its tab whereabouts is necessary

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,235 @@ from this dashboard.
+
+- :guilabel:`Payslip Name`: in the :guilabel:`Other Info` tab, type in a name for the payslip in
+  this field. The name should be short and descriptive, such as `(Employee Name) April 2023`. This
+  field is required.
```
```suggestion
  field is **required**.
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,235 @@ from this dashboard.
+- :guilabel:`Payslip Name`: in the :guilabel:`Other Info` tab, type in a name for the payslip in
+  this field. The name should be short and descriptive, such as `(Employee Name) April 2023`. This
+  field is required.
+- :guilabel:`Company`: in the :guilabel:`Other Info` tab, select the company the payslip applies to
```
```suggestion
- :guilabel:`Company`: select the company the payslip applies to
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,235 @@ from this dashboard.
+  this field. The name should be short and descriptive, such as `(Employee Name) April 2023`. This
+  field is required.
+- :guilabel:`Company`: in the :guilabel:`Other Info` tab, select the company the payslip applies to
+  using the drop-down menu in this field. This field is required.
```
```suggestion
  using the drop-down menu in this field. This field is **required**.
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,235 @@ from this dashboard.
+  field is required.
+- :guilabel:`Company`: in the :guilabel:`Other Info` tab, select the company the payslip applies to
+  using the drop-down menu in this field. This field is required.
+- :guilabel:`Close Date`: located in the :guilabel:`Other Info` tab, enter the date that the payment
```
```suggestion
- :guilabel:`Close Date`: enter the date that the payment
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,235 @@ from this dashboard.
+  (less-than/greater-than)` icons, navigate to the desired month and year.
+
+  Then, click on the desired date to select it.
+- :guilabel:`Date Account`: located in the :guilabel:`Other Info` tab, enter the date on which the
```
```suggestion
- :guilabel:`Date Account`: enter the date on which the
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,235 @@ from this dashboard.
+  Then, click on the desired date to select it.
+- :guilabel:`Date Account`: located in the :guilabel:`Other Info` tab, enter the date on which the
+  payslip should be posted in this field.
+- :guilabel:`Salary Journal`: this field, located in the :guilabel:`Other Info` tab, auto-populates
```
```suggestion
- :guilabel:`Salary Journal`: this field auto-populates
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,235 @@ from this dashboard.
+  payslip should be posted in this field.
+- :guilabel:`Salary Journal`: this field, located in the :guilabel:`Other Info` tab, auto-populates
+  after selecting an existing :guilabel:`Employee`. This field **cannot** be edited, as it is linked
+  to the *Accounting* application. This field is required.
```
```suggestion
  to the *Accounting* application. This field is **required**.
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,235 @@ from this dashboard.
+  after selecting an existing :guilabel:`Employee`. This field **cannot** be edited, as it is linked
+  to the *Accounting* application. This field is required.
+- :guilabel:`Accounting Entry`: if applicable, this field is automatically populated once the
+  payslip is confirmed. This field is not able to be modified.
```
```suggestion
  payslip is confirmed. This field **cannot** be modified.
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,235 @@ from this dashboard.
+- :guilabel:`Accounting Entry`: if applicable, this field is automatically populated once the
+  payslip is confirmed. This field is not able to be modified.
+- :guilabel:`Add an Internal Note...`: any note or reference message for the new entry can be typed
+  in the field.
```
```suggestion
  in this field.
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,235 @@ from this dashboard.
-:guilabel:`Cancel Payslip` button.
+When all the necessary information on the payslip is entered, click the :guilabel:`Compute Sheet`
+button. Upon doing so, all the information on the payslip is saved, and the :guilabel:`Salary
+Computation` tab auto-populates based on the information on the employee's contract or attendance
```
```suggestion
Computation` tab auto-populates, based on the information on the employee's contract or attendance
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,235 @@ from this dashboard.
-   banking information, if it is missing.
+   contact information. If there is no bank information, a payslip cannot be paid, and an error
+   appears when the :guilabel:`Make Payment` button is clicked. Banking information can be found in
+   the :ref:`Private Information <employees/private-info>` tab on the employee's card. Edit the
```
```suggestion
   the :ref:`Private Information <employees/private-info>` tab on the employee's card in the *Employees* app. Edit the
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,235 @@ from this dashboard.
+   contact information. If there is no bank information, a payslip cannot be paid, and an error
+   appears when the :guilabel:`Make Payment` button is clicked. Banking information can be found in
+   the :ref:`Private Information <employees/private-info>` tab on the employee's card. Edit the
+   employee card, and add banking information if it is missing.
```
```suggestion
   employee card, and add banking information, if it is missing.
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,235 @@ from this dashboard.
 All payslips
 ============
 
 To view all payslips regardless of status, go to :menuselection:`Payroll app --> Payslips --> All
```
```suggestion
To view all payslips, regardless of status, go to :menuselection:`Payroll app --> Payslips --> All
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,235 @@ from this dashboard.
 
 To view all payslips regardless of status, go to :menuselection:`Payroll app --> Payslips --> All
-Payslips`. Here, all payslips are organized by batch (in a default list view).
+Payslips`. The :guilabel:`Employee Payslips` page loads, displaying all payslips, organized by batch
```
```suggestion
Payslips`. The :guilabel:`Employee Payslips` page loads, displaying all payslips, organized by batch,
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,235 @@ from this dashboard.
+
+- :guilabel:`Draft`: the payslip is created, and there is still time to make edits, since the
+  amounts are not calculated.
+- :guilabel:`Waiting`: the payslip has been calculated and the salary details can be found in the
```
```suggestion
- :guilabel:`Waiting`: the payslip has been calculated, and the salary details can be found in the
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -190,27 +268,34 @@ Click on an individual payslip to view the details for that payslip on a separat
    :align: center
    :alt: Click on the Export All smart button to export all payslips to an Excel payslip.
 
+To export only select payslips, first select the payslips to be exported from the list. First, click
```
```suggestion
To export only select payslips, first select the payslips to be exported from the list. Then, click
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -190,27 +268,34 @@ Click on an individual payslip to view the details for that payslip on a separat
+To export only select payslips, first select the payslips to be exported from the list. First, click
+the checkbox to the left of each individual payslip to select it. As payslips are selected, a smart
+button appears in the top-center of the page, indicating the number of selected payslips. Then,
+click the :guilabel:`⚙️ (gear) Actions` icon in the top-center of the page, then click
```
```suggestion
click the :guilabel:`⚙️ (gear) Actions` icon in the top-center of the page, and click
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,166 +306,163 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
-Structure` and/or the :guilabel:`Department`, select a salary structure and/or department from the
-respective drop-down menus.
+Next, select the date range to which the batch applies. Click into one of the :guilabel:`Period`
+fields and a calendar pop-up window appears. From this calendar pop-up window, navigate to the
```
```suggestion
fields, and a calendar pop-up window appears. From this calendar pop-up window, navigate to the
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,166 +306,163 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
 
-Use the breadcrumb menu to navigate back to the individual batch detail page, or back to the list of
-all batches.
+- :guilabel:`New`: batches without any payslips added to them have a status of :guilabel:`New`. The
```
```suggestion
- :guilabel:`New` status: batches without any payslips added to them have a status of :guilabel:`New`. The
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,166 +306,163 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
-If the batch is a credit note, check the box next to :guilabel:`Credit Note`. Then, in the
-:guilabel:`Generation Date` field, select the date the payslips should be generated from a calendar
-pop-up window. This generated date is reflected in the accounting journal entries.
+  .. image:: payslips/batch-new.png
+     :align: center
+     :alt: A batch with a status of new, with the available buttons highlighted.
```
I believe this, and the other status-related images in this section of the doc would be more beneficial to reader if placed *between* the status definition and the subsequent buttons/options that appear for that specific status.

So, for this it would be 

New: ,,,,,

image

- Add Payslips: ....

and so on.

Please apply this suggestion for the remainder of screenshots used in this section. Thanks! 👍

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,166 +306,163 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
 
-Lastly, in the :guilabel:`Company` field, select the company these payslips are written against.
+- :guilabel:`Confirmed`: batches that have been created and have payslips in them, but the payslips
+  have *not* been processed have a status of :guilabel:`Confirmed`. The following two button options
```
```suggestion
  have *not* been processed, have a status of :guilabel:`Confirmed`. The following two button options
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,166 +306,163 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
 
-If a specific :guilabel:`Salary Structure` or :guilabel:`Department` needs to be specified for the
-batch, select them from the corresponding drop-down menus.
+  - :guilabel:`Create Payment Report`: click the :guilabel:`Create Payment Report` button and a
```
```suggestion
  - :guilabel:`Create Payment Report`: click the :guilabel:`Create Payment Report` button, and a
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,166 +306,163 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
+     :alt: A batch with a status of paid, with the available buttons highlighted.
 
-Once the payslips have been successfully generated, the screen returns to the payslip batch form.
+On the batch detail page, the individual payslips in the batch are accessible via the
```
```suggestion
On the batch detail page, the individual payslips in the batch are accessible, via the
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,166 +306,163 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
 
-When done, click the :guilabel:`Confirm` button to confirm the information, or click
-:guilabel:`Cancel` to discard.
+Commissions are paid to employees in Odoo using warrant payslips.
```
```suggestion
Commissions are paid to employees in Odoo using *warrant payslips*.
```

ksc-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,166 +306,163 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
 reveal calendar pop-up windows. On these calendar pop-up windows, select the desired period for
 which the payslips are being generated. Using the :guilabel:`< (left)` and :guilabel:`> (right)`
-arrows, navigate to the correct month, and click on the date to select it.
+arrows to navigate to the correct month, and click on the date to select it.
```
```suggestion
arrow icons, navigate to the correct month, and click on the date to select it.
```

ksc-odoo reviewed (approved):
Hey @larm-odoo -- thank you for implementing all the suggested edits from my first Final Review so quickly. I really appreciate it. Now, while there is over 40 new suggested edits, they are all relatively simple/minor, and I trust you'll be able to implement them all without any issue. So, with that in mind, I'll go ahead and approve this now. But, please make all the necessary adjustments that I've left for ya 🙏 

Once it's ready, and everything's looking good, feel free to tag this for Tech Review. Thanks! 👍

larm-odoo commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,235 @@ from this dashboard.
+   If modifications to auto-populated fields are made, it is recommended to check with the
+   accounting department to ensure every entry that affects the *Accounting* application is correct.
+
+Worked days & inputs tab
```
I would typically agree, but with the new style standards, we were supposed to be using this format. I have changed all the headings to be in this case, so since I feel it is more in alignment with the guidelines, I am going to keep this as it is.

larm-odoo commented:
Thanks so much @ksc-odoo!

larm-odoo commented:
Hey @samueljlieber - this is ready for a tech review =)

samueljlieber commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -221,166 +304,163 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
-to which the batch applies, by clicking the :guilabel:`▼ (drop-down arrow)` icon in the
-:guilabel:`Period` fields, which reveals a calendar pop-up window. From this calendar pop-up window,
-navigate to the correct month, and click on the corresponding day for both the start and end dates.
+   - :guilabel:`Generate Payslips`: after payslips have been added to the batch, click the
+     :guilabel:`Generate Payslips` button to process the payslips and create individual payslips in
+     the database.
```
If this bullet (:guilabel:\`Generate Payslips\`:) is intended to be a sub-bullet to (:guilabel:\`Add Payslips\`:), please indent it by another space, otherwise please scoot it back a space to align with its parent bullet point
```suggestion
    - :guilabel:`Generate Payslips`: after payslips have been added to the batch, click the
      :guilabel:`Generate Payslips` button to process the payslips and create individual payslips in
      the database.
```

samueljlieber reviewed (changes requested):
Hi @larm-odoo, great job on this PR! Your improvements to this doc look great. I have a few corrections and suggestions, please see below and tag me for another look once these have been addressed. Thank you!

Regarding the [headings casing conversation](https://github.com/odoo/documentation/pull/7740/files#r1525291255), I prefer having the heading casing match the UI when it is on-screen in the active instruction. Similar to how in the **Batches** section, you first instructed the user to navigate to the "Payslips Batches" page, and then in the subsequent sections (**Create a new batch**, **Process a batch**, and **Generate warrant payslips**) you were able to use the :guilabel:\`Payslips Batches\` because you already gave the active instruction in the top-level section.

It is the same idea for the heading casing, as long as that heading text is part of the active instruction, it can be capitalized to match the UI. 

Let me know your thoughts, and tagging @StraubCreative here if he has differing thoughts on the matter. 

Overall just a few more small improvements to be made! Thank you @larm-odoo :)

larm-odoo commented:
Thank you @samueljlieber! I conflated the buttons/screen text with GUI rules in those sections you caught- the changes all make sense to me, so thanks for catching that. =)  All changes are done, so this is ready for you again. 

As for the heading text matching the UI- I agree to have @StraubCreative decide. I know I have done them both ways, and my most recent PR I made them all lower case- and I don't have too much of a preference, so I don't mind which way we decide.

samueljlieber commented on content/applications/hr/payroll/payslips.rst:
```diff
@@ -28,159 +30,233 @@ from this dashboard.
-along with a PDF copy of the payslip.
+create the payslip.
+
+Then, a confirmation pop-up window appears, asking `Are you sure you want to proceed?`. Click
+:guilabel:`OK` to confirm.
```
On-screen text needs a :guilabel:
```suggestion
Then, a confirmation pop-up window appears, asking :guilabel:`Are you sure you want to proceed?`.
Click :guilabel:`OK` to confirm.
```

samueljlieber reviewed (approved):
Hi @larm-odoo, thank you for implementing my suggestions– everything looks good. Nice job! I found one more small correction, but I am approving now. Feel free to tag ZST :) thank you!

larm-odoo commented:
This is ready for a final review form you, @StraubCreative =)

StraubCreative reviewed (approved):
Approved, thanks @larm-odoo 👍 
...
@robodoo r+
